gcalbot/gcalbot/util.go

```diff
@@ -152,3 +156,16 @@ func MinutesBeforeString(minutesBefore int) string {
 		return fmt.Sprintf("%d minutes", minutesBefore)
 	}
 }
+
+func GetConvHelpText(channel chat1.ChatChannel, htmlFormat bool) string {
+	if channel.MembersType == "team" {
+		if htmlFormat {
+			return fmt.Sprintf(
+				`Configure Google Calendar notifications for a channel in the team <a target="_blank" href="https://keybase.io/team/%s">%s</a>`,
```

Comment:
let's leave the link out, people might not be logged in on the website and it would be confusing if it said not found